platform-api/src/resources/openapi.yaml

```diff
@@ -413,6 +413,35 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /apis/{apiId}/gateways:
```

Comment:
We should ideally have something like
  /apis/{apiId}/deployments:  
which will return deployment object list.
In each deployment object, we will have the deployed gateway, deployedTime, deploymentStatus (active/failed), etc. In the current model, we'll be limited to the gateway list only.
Shall we track this and re-implement as a separate task?